src/rules/use-tokens.js

```diff
@@ -118,7 +128,26 @@ const checkColor = (node, context) => {
       message: `Use the following Backpack token instead: ${
         expectedToken.name
       }`,
-      fix: fixer => fixer.replaceText(value, expectedToken.name),
+      fix: fixer => {
+        const { options: userOptions } = context;
+        const options = _.merge({}, BASE_CONFIG, userOptions[0] || {});
+        const tokensPkg = options.tokensPackage[options.platform];
+
+        const importDefinition = getImportDefinition(
```

Comment:
nitpic: We don't need to construct this value if we have `!options.autoImport`.

Reply:
I'll address this on #10